src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,19 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.14.4
+
+This is a bugfix release:
+
+* Fix stale IPtable rules along with globalIP leak which can sometimes happen as part of stress testing.
+* Handle out of order remote endpoint notifications properly in various route-agent handlers.
+* Ensures that reverse path filtering setting is properly applied on the relevant network interfaces.
+* Improve public IP resolver logging.
+* Log connection status in Submariner gateway pod.
+* Properly disable gateway controller in ServiceDiscovery only deployments.
+* Gather ipset details from all the nodes as part of `subctl gather`.
+* Support gathering operator logs even when it deployed with label other than `name=submariner-operator`.
```

Comment:
This is related to https://github.com/submariner-io/subctl/issues/490 which only pertained to the Submariner addon so I don't think we need to release not it.

Reply:
So, we only need the items from https://github.com/submariner-io/submariner/tree/release-0.14/release-notes after all, right?

Reply:
It seems so

Reply:
@sridhargaddam @dfarrell07  can you take a look? This list seems incomplete to me.